conf/nginx.conf.template

```diff
@@ -300,12 +300,14 @@ http {
     # To solve this, a specific location has been added for that will force a
     # real request by using a proxy_pass to a full url.
     #
-    location ~ ^/force_real_request/(.*)$ {
+    location /force_real_request/ {
       # Not exposed to external requests.
       #
       internal;
 
-      proxy_pass http://127.0.0.1/$1$is_args$args;
+      rewrite ^/force_real_request(.*)$ $1 break;
+
+      proxy_pass http://127.0.0.1;
```

Comment:
by doing so, nginx does not "normalize" the URI sent to GCS. We keep the encoding for the space and the GCS backend does not refuse it.